src/main/scala/io/sdkman/changelogs/jmc/OpenJDKJmcMigrations.scala

```diff
@@ -0,0 +1,50 @@
+package io.sdkman.changelogs.jmc
+
+import com.github.mongobee.changeset.{ChangeLog, ChangeSet}
+import com.mongodb.client.MongoDatabase
+import io.sdkman.changelogs.{Linux64, MacOSX, Version, Windows, _}
+
+@ChangeLog(order = "029")
+class OpenJDKJmcMigrations {
+
+  @ChangeSet(order = "001", id = "001_add_jmc_candidate", author = "den1ska")
+  def migration001(implicit db: MongoDatabase) = {
+    Candidate(
+      candidate = "jmc",
+      name = "JDK Mission Control",
+      description =
+        "JDK Mission Control 7 (JMC) is an advanced set of tools that enables efficient and detailed data analysis, and delivers advanced, unobtrusive Java monitoring and management. JMC provides sections for common analysis areas such as code performance, memory, and latency.",
+      websiteUrl = "https://jdk.java.net/jmc/",
+      distribution = "UNIVERSAL"
```

Comment:
This looks more like `PLATFORM_SPECIFIC` to me. wdyt?

Reply:
@marc0der I would think too. Also there are multiple vendors involved as in the Java JDKs...

Reply:
This is a very old PR. JMC is long since available on SDKMAN, all though nobody seems to be using it.